src/task.c

```diff
@@ -721,26 +785,44 @@ void jl_init_root_task(void *stack, size_t ssize)
     ptls->current_task->ssize = ssize;
     ptls->current_task->stkbuf = stack;
 #endif
-    ptls->current_task->started = 1;
-    ptls->current_task->parent = ptls->current_task;
-    ptls->current_task->current_module = ptls->current_module;
+#ifdef JULIA_ENABLE_PARTR
+    ptls->current_task->storage = jl_nothing;
+    ptls->current_task->args = jl_nothing;
+    ptls->current_task->mfunc = NULL;
+    ptls->current_task->rargs = jl_nothing;
+    ptls->current_task->mredfunc = NULL;
+    ptls->current_task->next = NULL;
+    ptls->current_task->parent = NULL;
```

Comment:
This needs to point to a Task.

Reply:
`parent` can be made to point to `current_task`, but what about `next`? We need a way to determine the end of the list. Should I create a global dummy sentinel?

Reply:
You can use `nothing` as a sentinel, or leave it as NULL (though there is no way to set a field to NULL from julia).

Reply:
Currently we make the root task its own parent.

Reply:
Currently, the `parent` field is deprecated

Reply:
It's still referenced in the loading code, which was causing the sysimg build to fail.

Reply:
partr uses `parent` to identify the first task in the set of tasks created for a `parfor`.